bitrise.yml

```diff
@@ -22,6 +22,10 @@ stages:
       - check: { }
       - test: { }
       - run-instrumentation-tests: { }
+      - run-paymentsheet-instrumentation-tests: { }
+      - run-example-instrumentation-tests: { }
```

Comment:
doesn't `run-instrumentation-tests` already run all the tests? @jaynewstrom-stripe

Reply:
It did: https://github.com/stripe/stripe-android/pull/7373/files#diff-61d1ee831cb1df242161a7d23171914a1d5d628c7659b52db957d80a9394bd68R281

But I changed it to exclude all the ones run individually now.